lib/model.js

```diff
@@ -1394,14 +1394,14 @@ Model.prototype.count = function(options) {
  * @return {Promise<Object>}
  * @alias findAndCountAll
  */
-Model.prototype.findAndCount = function(findOptions) { // testhint options:1
```

Comment:
Yes, the hint was only there because the argument wasn't called `options`, so it needed the hint to tell it where to attach `.logging` to.

This is really why I'm submitting this PR - just to allow the hint to be removed, and so to avoid confusion.